algorithms/CPlusPlus/Strings/ReverseTheStringWordwise.cpp

```diff
@@ -0,0 +1,44 @@
+// Description :- Given a string, the task is to reverse the order of the words in the given string. 
+// Example :-
+// Input 1:
+//    A = "the sky is blue"
+// Input 2:
+//    A = "this is ib"
+// Output 1:
+//    "blue is sky the"
+// Output 2:
+//    "ib is this"
+
+
+// Time Complexity = O(N), Space Complexity = O(N)
```

Comment:
include the necessary headers
```suggestion
// Time Complexity = O(N), Space Complexity = O(N)

#include <iostream>
#include <string>
#include <vector>
```